src/parser/ui/BoringSpellValueText.tsx

```diff
@@ -2,12 +2,13 @@
  * A simple component that shows the spell value in the most plain way possible.
  * Use this only as the last resort, prefer a custom layout instead.
  */
+import Spell from 'common/SPELLS/Spell';
 import { SpellIcon } from 'interface';
 import { SpellLink } from 'interface';
 import { ReactNode } from 'react';
 
 interface Props {
-  spellId: number;
+  spellId: number | Spell;
```

Comment:
I think I'd rather we change this to a different name but that's not for this PR. We're a bit inconsistent in how spells vs spell IDs are handled.